guides/development.md

```diff
@@ -1,10 +1,12 @@
 # Development
 
-From the root folder:
+After cloning the repository, from the root folder:
 
-- Run `yarn` to install dependencies.
+- Run `yarn install` to install dependencies.
 
-- Run `yarn start` to start the development server for the package and for the example.
+- Run `yarn build:packages`
+
+- Run `yarn start:www` to start the development server for the package and for the example.
```

Comment:
actually this only start the website in apps/www
yarn start will start the packages and you can navigate to localhost:5420 to see it